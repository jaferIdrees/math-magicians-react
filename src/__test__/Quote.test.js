import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Quote component testing using snapshots', () => {
  it('Test Quote component renders', () => {
    const tree = renderer
      .create(<Quote />);
    expect(tree).toMatchSnapshot();
  });
});
