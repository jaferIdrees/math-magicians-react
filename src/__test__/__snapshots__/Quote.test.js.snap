// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component testing using snapshots Test Quote component renders 1`] = `
<div
  className="homePageContainer"
>
  <p
    className="homePageText"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. (William Paul Thurston)
  </p>
</div>
`;
